.github/dependabot.yml

```diff
@@ -7,6 +7,13 @@ updates:
     labels:
       - "skip changelog"
       - "tomcat 9"
+  - package-ecosystem: "maven"
+    directory: "/webapp-runner-10"
+    schedule:
+      interval: "monthly"
+    labels:
+      - "skip changelog"
+      - "tomcat 10"
```

Comment:
It might be worth adding an `ignore` section to both the v9 and v10 Dependabot configs to ignore major version releases of tomcat using `update-types` - otherwise Dependabot will try to update both to the latest release:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

Reply:
This is currently configured implicitly for `webapp-runner-9` (by commenting on the PR that dependabot should ignore that major version). Might be worth being explicit there, agreed 👍🏻 . Let's move that to a separate PR though as it affects not only this new version.